IBM/CAST
# Ufm inventory updates

# UFM inventory updates

## Purpose
Addresses issues opened in #336 and provides more user feedback in the standalone ufm inventory collection

## Approach
- adds more to CSM database functions to remove old records
- adds more user feedback in standalone inv collection program
  - lists new, updated, and removed number of records from database
  - this is to more easily see changes from last collection run.  

## How to Test

1. functionality tests have already been done. 
2. nothing more to test. 
3. may have to update older regression tests
4. should re run current regression to make sure this doesnt break anything. 

## Screenshots

Target area of improvement:
```
totalIBRecords 108
totalSwitchRecords 7
---
# ib inventory successful
...
# total ib inventory collected: 108
# new ib records inserted into database: 0
# old ib records updated in database: 108
# old ib records removed from the database: 0
---
# switch inventory successful
...
# total switch inventory collected: 7
# new switch records inserted into database: 0
# old switch records updated in database: 7
# old switch records removed from the database: 0
# old switch module records removed from the database because they were associated with the above removed switches: 0
---
# switch module inventory successful
...
# total switch module inventory collected: 64
# new switch module records inserted into database: 0
# old switch module records updated in database: 64
# old switch module records removed from the database: 1
[root@c650f03p41 sbin]# 

```
